### PR TITLE
Add support to display running alerts, add tabs & Rogerize it.

### PR DESCRIPTION
**Pull request by vmahedia**
1. Add JQuery UI 
2. Add Function to get running alert report from alertd 
3. Add tab support 
4. Add table to display the alerts
5. Add Roger Icon - Rogerize it !

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
     # Configure the server object
     server.socket_host = "0.0.0.0"
-    server.socket_port = 8080
+    server.socket_port = 8888
```
Comment (ankanm) [inline server.py]: Why?
Also, if this is done other files like Dockerfile and the template in roger-cli need to be changed too.

File: dash/templates/index.html [hunk 2]
```diff
@@ -31,24 +92,82 @@
...
+    var alert_array = [];
+    var alert_table = null;
+    alert_table = new google.visualization.Table(
+      document.getElementById('alert_notifications'));
+    var alert_data_table = new google.visualization.DataTable();
+    alert_data_table.addColumn('string', 'Latest Alert');
+    alert_data_table.addColumn('string', 'Time');
+    $.ajax({
+        dataType: 'JSON',
+        url: "http://localhost:8008/stats/running-alert-report",
```
Comment (ankanm) [inline dash/templates/index.html]: Let's add an env var and render it using jinja.

File: server.py (hunk 1 above)
Comment (vmahedia) [inline server.py]: Port conflict with marathon UI while running locally.

**Comment by ankanm** [comment]
LGTM

**Pull request merged**